tools/changelog.sh

```diff
@@ -0,0 +1,7 @@
+folder=$(mktemp -d)
+git log | node tools/changelog >$folder/CHANGELOG.md
```

Comment:
`changelog.js` should move to `lib` because only outdated stuff is kept in `tools` === "garbage bin"